mobile/examples/object_detection/ios/Podfile.lock

```diff
@@ -0,0 +1,22 @@
+PODS:
```

Comment:
Add this to the .gitignore?

Reply:
oh I saw in the tensorflow examples, all of their `podfile.lock` is left there. so figured it might be okay.